### DISCUSSION TITLE
Support for a TS-XW w/ T3PA pedals.

### DISCUSSION THREAD

**Discussion by AngelaPimax** [berarma/oversteer#159] [Q&A]
Hopefully you can help me out! 

I see you said just to contact the page if your hardware is not supporte.d I run a TS-X2 w/ T3PA pedals. If you need any additional information just post here. 

I really appreciate your time and I love this application :-)

**Comment by berarma** [comment]
I've copied your request to issue #161 so I keep all requests in one place.
